uaclient/messages.py

```diff
@@ -466,6 +466,29 @@ class TxtColor:
     + BASE_UA_URL,
 )
 
+MAGIC_ATTACH_TOKEN_EXPIRED_OR_ACTIVATED = FormattedNamedMessage(
+    "magic-attach-token-expired-or-activated",
+    (
+        "The magic attach token: {magic_token} is already "
+        "activated or expired."
+    ),
+)
+
+MAGIC_ATTACH_EXPIRED_TOKEN = FormattedNamedMessage(
+    "magic-attach-token-expired",
+    "The magic attach token: {magic_token} has expired or never existed.",
```

Comment:
I'd recommend against printing the token in all these cases. There's nothing the user can do with this information, and it just increases the chances of toking ending up in logs (even though they'd be invalid tokens).

Reply:
Good point, I will remove this